src/Raven.Client/Documents/DocumentStore.cs

```diff
@@ -268,21 +268,31 @@ public override IDisposable DisableAggressiveCaching(string database = null)
         /// <summary>
         /// Subscribe to change notifications from the server
         /// </summary>
-        public override IDatabaseChanges Changes(string database = null)
+        public override IDatabaseChanges Changes(string database = null, string nodeTag = null)
```

Comment:
You need to create a separate `Changes` method overload to not brake the binary compatibility

Reply:
done